src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -0,0 +1,115 @@
+
+import { useLoginForm } from '../../../hooks';
+
+export default function LoginForm(): React.ReactElement {
+  const { formik, success, error } = useLoginForm();
+
+  return (
+      <form onSubmit={formik.handleSubmit}>
+        {success ? <h2 id="success">SUCCESS!</h2> : null}
+        {error ? <h2 id="error">ERROR!</h2> : null}
+        <HeadingMedium
+          color="fontColor"
+          marginBottom="scale600"
+        >
+          Please sign in
+        </HeadingMedium>
+
+        <Block>
+          <FormControl
+            label={() => 'Email address'}
+            error={formik.errors.username}
+          >
+            <Input
+              overrides={{
+                Root: {
+                  props: {
+                    'data-testid': 'username',
+                  },
+                },
+                Input: {
+                  style: ({ $theme }) => ({
+                    backgroundColor: `${$theme.colors.inputBackground}`,
+                    color: `${$theme.colors.fontColor}`,
+                    caretColor: `${$theme.colors.fontColor}`,
+                  }),
+                },
+              }}
+              name="username"
+              placeholder="Email address"
+              value={formik.values.username}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              autoComplete="off"
+            />
+          </FormControl>
+        </Block>
+        <Block>
+          <FormControl label={() => 'Password'} error={formik.errors.password}>
+            <Input
+              overrides={{
+                Root: {
+                  props: {
+                    'data-testid': 'password',
+                  },
+                },
+                Input: {
+                  style: ({ $theme }) => ({
+                    backgroundColor: `${$theme.colors.inputBackground}`,
+                    color: `${$theme.colors.fontColor}`,
+                    caretColor: `${$theme.colors.fontColor}`,
+                  }),
+                },
+              }}
+              name="password"
+              placeholder="Password"
+              value={formik.values.password}
+              onChange={formik.handleChange}
+              onBlur={formik.handleBlur}
+              autoComplete="off"
+              type="password"
+            />
+          </FormControl>
+        </Block>
+        <Block style={{ marginTop: '1rem', marginBottom: '1rem' }}>
+          <Checkbox
+            name="rememberMe"
+            checked={formik.values.rememberMe}
+            onChange={formik.handleChange}
+            labelPlacement={LABEL_PLACEMENT.right}
+            overrides={{
+              Checkmark: {
+                style: ({ $theme }) => ({
+                  backgroundColor: `${$theme.colors.inputBackground}`,
+                  border: `1px solid ${$theme.border.borderColor}`,
+                  borderRadius: '0.25em',
+                }),
+              },
```

Comment:
Override globally or create custom component with the overrides. A page should not create overrides for primitive components

Reply:
Created Custom Component